Finmer.Editor/FormMain.cs

```diff
@@ -282,6 +282,23 @@ private bool SaveAs()
             if (dlgSave.ShowDialog() != DialogResult.OK)
                 return false;
 
+            else 
+            {
+                // Pull the directory we're saving in, see if there's any other finmer projects already there.
+                string save_path = Path.GetDirectoryName(dlgSave.FileName);
+                string[] file_entries = Directory.GetFiles(save_path, "*.fnproj");
+                if (file_entries.Length >= 1)
+                {
+                    // Alert user that saving here will delete unusued json files in the directory, have them acknowledge that fact before continuing.
```

Comment:
unusued -> unused

Reply:
One day I will get that tic with u's and other consonants corrected, but that's not today...